packages/core/src/browser/label-provider.ts

```diff
@@ -73,7 +79,7 @@ export class DefaultUriLabelProviderContribution implements LabelProviderContrib
         return 0;
     }
 
-    getIcon(uri: URI): MaybePromise<string> {
+    getIcon(uri: URI): string {
```

Comment:
the same as https://github.com/eclipse-theia/theia/pull/6475/commits/2795c7ecf6644c2e0d823562fe59a70af522781b#r350290387